docs/_series/mvpphock.md

```diff
@@ -0,0 +1,70 @@
+---
+name: MVPPHOCK
+author: BOYSCLUBX
+image: https://i.imgur.com/CUjRCp3.gif
+date: 2022-03-03
+description: NBA Most Valuable Player (MVP) award is presented to the best player of the NBA regular season. The MVPPHOCK grail holder will get 14 Sub Assets of NBA MVP player, after all grail sold/ distributed.
+subs: 
+  -
+    name: MVPPHOCK.ALLENIVERSON
```

Comment:
Sub assets names MUST be without the main token name. 

`MVPPHOCK.ALLENIVERSON` should be just `ALLENIVERSON`